.gitignore

```diff
@@ -6,3 +6,4 @@
 venv/
 .pytest_cache/
 **__pycache__/
+*.csv
```

Comment:
The pattern `*.csv` will ignore all CSV files in the repository. If you intend to keep some CSV files under version control, consider specifying a more precise pattern or removing this line.